Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -79,6 +100,20 @@ public final class SwiftLanguageServer: ToolchainLanguageServer {
     response: SKResponseDictionary,
     for snapshot: DocumentSnapshot)
   {
+    // Disable diagnostics for blacklisted URLs. Some editors (like Visual Studio Code)
```

Comment:
Let's fold all the documentation onto the doc-comment string for the function rather than duplicate it here.